### DISCUSSION TITLE
Future development

### DISCUSSION THREAD

**Discussion by martinspgronli** [petrobras/ccp#42] [General]
Hi! I see that there are some development going on here. I was wondering if there are plans to extend this code to include multi-stage compressors? Or will the focus be on straight through compressors and back-to-back compressor with 2 sections?

Best,
Martin

**Comment by raphaeltimbo** [comment]
Hi @martinspgronli!

It seems like you're referring to the compressor.py module. This module is specifically designed for modeling compressors, with a focus on factory performance testing. In this context, we need to have precise control over the calculation of leakage values in the balance piston/division wall.

Currently, we only have support for straight-through and back-to-back compressors in the module. These two configurations represent the majority of the compressors used at Petrobras, considering a single casing (but we are always open to accepting contributions to the module).

However, if you are not so worried about measuring/calculating leakages, you can still represent a compressor with multiple sections by instantiating different Impeller objects. Each Impeller object can represent one section of the compressor.

Please let me know if you have any further questions.

**Reply by martinspgronli** [reply]
Hi @raphaeltimbo

Thanks for the answer!

Best, 
Martin
